packages/hasura-auth-js/auth.typedoc.json

```diff
@@ -1,8 +1,12 @@
 {
   "$schema": "https://typedoc.org/schema.json",
-  "entryPoints": ["src/index.ts"],
+  "entryPoints": [
+    "src/hasura-auth-client.ts"
```

Comment:
I don't think we should change this. `src/index.ts` is the file exporting the relevant types. If we change this to `src/hasura-auth-client.ts`, then TypeDoc will complain about missing type exports.

e.g:
```
./src/types/client.ts:91:0 - warning SendVerificationEmailParams is referenced by HasuraAuthClient.sendVerificationEmail.sendVerificationEmail.__namedParameters but not included in the documentation.
./src/types/client.ts:96:0 - warning ChangeEmailParams is referenced by HasuraAuthClient.changeEmail.changeEmail.__namedParameters but not included in the documentation.
./src/types/client.ts:101:0 - warning DeanonymizeParams is referenced by HasuraAuthClient.deanonymize.deanonymize.params but not included in the documentation.
./src/types/client.ts:119:0 - warning OnTokenChangedFunction is referenced by HasuraAuthClient.onTokenChanged.onTokenChanged.fn but not included in the documentation.
./src/types/client.ts:117:0 - warning AuthChangedFunction is referenced by HasuraAuthClient.onAuthStateChanged.onAuthStateChanged.fn but not included in the documentation.
```

Did you experience any problems with `src/index.ts`?

Reply:
Yes I did, you can reproduce with
```sh
pnpm run docgen
pnpm run build:docs
```

It seems the generated markdown files are not always correct.
For instance `customSet` in `src/types/local-storage.ts`

Reply:
Yes, the `|` character drives the markdown crazy. I'll prepare a fix for that problem soon. I don't want to lose this many types from our docs.